### ISSUE TITLE
 can lazylibrarian work over a local network? 

### ISSUE THREAD

**Issue by justagame** [DobyTang/LazyLibrarian#746]
To help with identifying and fixing issues, please include as much information as possible, including:
### Relevant debug log with api keys and any passwords redacted

Please note - usually a single line of log is not sufficient. The lines just before the error occurs can give useful context and greatly assist with debugging.

**Comment by justagame** [comment]
sorry if this is a duplicate question... can lazylibrarian work over a local network? 
I have LL installed on my main server (ubuntu 16 something), my calibre library on a Synology NAS drive and nzbget runs on a raspberry pi 3 with openVPN.  For the life of me, I can't figure out how to a) point LL at my calibre database and b) while books can be snatched, they never are seen after downloading and being saved in my .../nzbget/completed directory by LL for postprocessing....

the debug log just returns the following:
2017-04-24 15:43:30	INFO	Scanning ebook directory: /
2017-04-24 15:43:30	WARNING	Destination dir [/192.168.0.240/volume1/SynMediaFiles/Books/My-Books] not found, using /
2017-04-24 15:43:27	INFO	Debug log display ON, loglevel is 2

LL version as of this am: Repo: https://github.com/dobytang/lazylibrarian : Branch: master
Current Version: ed0e79cfbe0eb90b87d1af317a25bb9e17552046 : Latest Version: ed0e79cfbe0eb90b87d1af317a25bb9e17552046
version 19 database
goodreads

**Comment by justagame** [comment]
sorry, also - the nas calibre book directory permissions and ownership are set to 777 and myusername:myusername

**Comment by philborman** [comment]
Short answer is yes, it can work over a network.
Longer answer is ...but it can be challenging setting up the shared folders.
Lazylibrarian needs to be able to access the nzbget download folder and the calibre books folder. How are you sharing them?  nfs is probably best/easiest as all three machines are linux based.

**Comment by Code-Slave** [comment]
I do this by mounting each with NFS or smb. I have tried both and they work. You just have to be very careful with paths for the various things (downloads/calibre). You might have a problem with mounting your nzbget server as if its vpn 24/7 then you may have to adjust routing or add a second adapter

**Comment by justagame** [comment]
Sorry for the delay - finally figured out the problems.....

Now using Repo: https://github.com/dobytang/lazylibrarian : Branch: master
Current Version: 5abff5a509d2caf317625b0455c614090bd9acc1 : Latest Version: 5abff5a509d2caf317625b0455c614090bd9acc1

- changed to the bookstrap interface just by selecting it from the Interface|interface: options makes reading the screen items much, much easier
- using SMB, so I made the following changes (Processing|Folders|eBook Library Folder) to point to the calibre library folder "/mnt/SynMediaFiles/Books/MyBooks".  The IP address and volume1 wasn't needed  as it wasn't networking that way.... 
- In addition, for some quirky reason, only after I renamed the library "MyBooks" from "My-Books"  did it work correctly.
- Also found out that the Downloaders|Download Directory has to include the category name - so after changing that to "/mnt/SynMediaFiles/OAMS/Downloads/nzbget/completed/**Books**"
- and finally, used "whereis calibredb" to find out where the calibre command line program (aka calibredb) was located on my server.  This provided two options:  /usr/bin/calibredb and /opt/calibre/bin/calibredb  .  Because my calibre library and lazylibrarian books are in the same location, I wanted LL to process moving the downloaded books from (see above) the download directory to the shared calibre-LL library.  Of course, the first one I used "/opt/calibre/bin/calibredb" gave me this error: "error while loading shared libraries: libcalibre-launcher.so: cannot open shared object file: No such file or directory".  When directly logged into a terminal on the server, running "/opt/calibre/bin/calibredb list" gives the exact same error.  But running "/usr/bin/calibredb list" worked fine, generating a list of my books.  So I plugged "**/usr/bin/calibredb**" as the location for Processing|Folders|Calibredb import program: and it works perfectly!

Now, my next request is, when books are added to calibre, can they be converted automatically into the format I have set in my preferences for additions?
![image](https://user-images.githubusercontent.com/27988823/26997160-3a91842a-4d2c-11e7-8835-ea7d2ab7adc1.png)
